assets/src/stories-editor/components/with-meta-block-edit.js

```diff
@@ -64,6 +66,21 @@ class MetaBlockEdit extends Component {
 		}
 	}
 
+	/**
+	 * Checks if the block's content has loaded, relevant for calculating font size of author block.
+	 *
+	 * @return {boolean} If content has loaded.
+	 */
+	contentHasLoaded() {
+		const { name, blockContent } = this.props;
+		if ( 'amp/amp-story-post-author' !== name ) {
+			return true;
+		}
+
+		const anonymousUserName = __( 'Anonymous', 'amp' );
+		return anonymousUserName.toLowerCase() !== blockContent.toLowerCase();
```

Comment:
Not sure if this might cause issues in some case, perhaps there's a better way to check if the content has loaded. Thoughts?

Reply:
That seems a bit overly complicated.

Why not just return an `isLoading` prop or so in the `withSelect` part further down?
The component itself shouldn't be concerned with deciding that.

I suppose `isLoading` could mean either `getEditedPostAttribute()` or `getAuthors()` is returning `undefined`.

Reply:
Yep, that makes 100x more sense. 🤦‍♀